File: php/utility/FeatureAdd.php
<?php
declare(strict_types=1);

// KonkanRailwayLivePosition SDK utility: feature_add

class KonkanRailwayLivePositionFeatureAdd
{
    public static function call(KonkanRailwayLivePositionContext $ctx, mixed $f): void
    {
        $ctx->client->features[] = $f;
    }
}

File: go/utility/feature_add.go
package utility

import "github.com/voxgig-sdk/konkan-railway-live-position-sdk/go/core"

func featureAddUtil(ctx *core.Context, f core.Feature) {
	client := ctx.Client
	features := client.Features

	client.Features = append(features, f)
}
